tests/lib/src/common_utils.h

```diff
@@ -9,105 +9,182 @@
 using namespace InferenceEngine;
```

Comment:
Perhaps it is worth removing this namespace, since now it is not prevails here

Reply:
Fixed